pgfinder/pgio.py

```diff
@@ -23,6 +24,21 @@
 LOGGER = logging.getLogger(LOGGER_NAME)
 
 
+def mass_libraries() -> Dict:
+    """
+    Loads built-in mass libraries from csv files as a dictionary.
+
+    `.csv` is chosen for consistency of data storage over the project.
+
+    Returns
+    -------
+    dict
+        A dictionary of filenames and paths to read from
+    """
+    mass_lib_path = Path(sys.modules[__package__].__file__).parent / "masses"
```

Comment:
Should try to avoid hard coded variables such as `/ "masses"` here, make it an option with the default value instead.

Reply:
Normally I'd obviously agree, but these are paths to assets bundled in the package, so I'm not sure if that would add so much utility here? It's not a function I think the user would typically ever call?

I do think it could be worth replacing the "masses" with a constant in the `__init__.py`? It's in the same category as the `"config/parameters.yaml"` there I think :)

What do you think?